tests/migration/migration.go

```diff
@@ -1882,15 +1880,18 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 					"config1": "value1",
 					"config2": "value2",
 				}
-				secret_data := map[string]string{
-					"user":     "admin",
-					"password": "community",
-				}
+
+				secret := libsecret.New(secretName, map[string][]byte{
+					"user":     []byte("admin"),
+					"password": []byte("community"),
+				})
+				_, err := kubevirt.Client().CoreV1().Secrets(testsuite.GetTestNamespace(nil)).Create(context.Background(), secret, metav1.CreateOptions{})
```

Comment:
Why have you changed the order of the objects creation? The secret is not created before the config-map.
Please reason for it in the commit.